tox.ini

```diff
@@ -32,6 +26,8 @@ basepython =
     python3.6
 commands =
     coverage run -m zope.testrunner --test-path=.
+    python -c 'import os, subprocess; env = os.environ.copy(); env["PURE_PYTHON"] = "1"; subprocess.check_call("coverage run -a -m zope.testrunner --test-path=.", shell=True, env=env)'
+    python -c 'import os, subprocess; env = os.environ.copy(); env["PURE_PYTHON"] = "0"; subprocess.check_call("coverage run -a -m zope.testrunner --test-path=.", shell=True, env=env)'
```

Comment:
Uhh, this is for cross-platformness of some kind?

Because `env PURE_PYTHON=1 coverage run -a -m ...` would be shorter and more readable on any OS that has /usr/bin/env.  Google tells me it exist on MacOS, as well as on Linux.  Do we care about people using `tox` on Windows?

Suggestion: `import os, subprocess; subprocess.check_call(..., env=dict(os.environ, PURE_PYTHON="0"))`  to make the oneliners shorter.

Reply:
> Uhh, this is for cross-platformness of some kind?
Right.

> Because env PURE_PYTHON=1 coverage run -a -m ... would be shorter and more readable on any OS that has /usr/bin/env. Google tells me it exist on MacOS, as well as on Linux. Do we care about people using tox on Windows?

Using `env` didn't occur to me. Doing so would require whitelisting the command to avoid the warning from tox about using an executable outside the environment, though.

> Suggestion: import os, subprocess; subprocess.check_call(..., env=dict(os.environ, PURE_PYTHON="0")) to make the oneliners shorter.

Oh, I like that. Thank you. I don't think I've ever actually used (or even come across!) that `dict` constructor.